examples/connected-car-cluster-sharding/akka-connected-car-streamlet/src/test/resources/application.conf

```diff
@@ -0,0 +1,5 @@
+akka {
+  actor {
+    provider = "cluster"
+  }
+}
```

Comment:
I have implemented the cloudflow-akka-testkit TestContext for shardedPlainSource and shardedSourceWithCommittableContext as well as created an example in the connected-car-cluster-sharding example.  The only thing that I don't like is that I had to manually add the `akka.actor.provider = "cluster"` to the test application.conf.

My concern is that you could potentially have multiple different tests in a project for both clustered and non-clustered streamlets.  Should I create a helper function that will create a clustered Akka system just for tests or something like that?

Reply:
This should be at least documented, that you need to do this for testing. I think that would be good enough for now.